src/components/WalletModal/index.tsx

```diff
@@ -144,14 +144,16 @@ export default function WalletModal({
   const previousWalletView = usePrevious(walletView)
 
   const [pendingConnector, setPendingConnector] = useState<Connector | undefined>()
-  // Need to pass network as a default case because useSelectedError requirse a connector
-  const pendingError = hooks.useSelectedError(pendingConnector || network)
+  // Need to pass infura as a default case because useSelectedError requirse a connector
```

Comment:
```suggestion
  // Need to pass infura as a default case because useSelectedError requires a connector
```

Reply:
thank you, added here https://github.com/Uniswap/interface/pull/3759/commits/3e61704aed320fdf261ce2633a6861865dfa8e76